doc/tutorial/search/index.md

```diff
@@ -158,86 +158,138 @@ The following hits were found:
 **Expected output:**
 ```console
-There are 12 hits.
-At position 1: GCT
-At position 5: TCT
-At position 12: GAT
-At position 23: GCC
-At position 36: GAT
-At position 41: GCT
-At position 57: ACT
-At position 62: GCC
-At position 75: GCG
-At position 77: GCT
-At position 83: GCA
-At position 85: ACT
+[<query_id:0, reference_id:0, reference_pos:1>,
+ <query_id:0, reference_id:0, reference_pos:5>,
+ <query_id:0, reference_id:0, reference_pos:12>,
+ <query_id:0, reference_id:0, reference_pos:23>,
+ <query_id:0, reference_id:0, reference_pos:36>,
+ <query_id:0, reference_id:0, reference_pos:41>,
+ <query_id:0, reference_id:0, reference_pos:57>,
+ <query_id:0, reference_id:0, reference_pos:62>,
+ <query_id:0, reference_id:0, reference_pos:75>,
+ <query_id:0, reference_id:0, reference_pos:77>,
+ <query_id:0, reference_id:0, reference_pos:83>,
+ <query_id:0, reference_id:0, reference_pos:85>]
 ```
 \endsolution
 
 ## Which hits are reported?
 
-Besides the error configuration, you can define which hits should be reported:
+Besides the max error configuration, you can specify the scope of the search algorithm. This means that you can control
+which hits should be reported by the search.
+
+### Hit configuration
+
+To do so, you can use one of the following \ref search_configuration_subsection_hit_strategy "seqan3::search_cfg::hit_*"
+configurations:
 
 - seqan3::search_cfg::hit_all: Report all hits that satisfy the (approximate) search.
 - seqan3::search_cfg::hit_single_best: Report the best hit, i.e. the *first* hit with the lowest edit distance.
 - seqan3::search_cfg::hit_all_best: Report all hits with the lowest edit distance.
 - seqan3::search_cfg::hit_strata: best+x strategy. Report all hits within the x-neighbourhood of the best hit.
 
-Any hit configuration element is appended to the error configuration by using the `|`-operator:
-\snippet doc/tutorial/search/search_small_snippets.cpp hit_best
+Opposed to the max error configuration, which allows a combination of the different error configuration objects, the hit
+configuration can only exists once within one search configuration. Trying to specify more than one hit configuration
+in one search configuration will fail at compile time with a static assertion.
+Sometimes the program you write requires to choose between different hit configurations depending on a user given
+program argument at runtime. To handle such cases you can also use the dynamic configuration seqan3::search_cfg::hit.
+This configuration object represents one of the four hit configurations mentioned previously and can be modified at
+runtime. The following snippet gives an example for this scenario:
+
+\snippet doc/tutorial/search/search_small_snippets.cpp hit_dynamic
+
+Note that the same rules apply to the dynamic as for the static hit configurations. Meaning, it can be added via the
+`|`-operator to the search configuration but cannot be combined with any other hit configuration.
```

Comment:
```suggestion
Note that the same rules apply to both the dynamic and static hit configuration. That is, it can be added via the
`|`-operator to the search configuration but cannot be combined with any other hit configuration.
```